art/art.py

```diff
@@ -125,28 +125,30 @@ def help_func():
     print(CLI_HELP)
 
 
-def aprint(artname, number=1):
+def aprint(artname, number=1,space=1):
     """
     Print 1-line art.
 
     :param artname: artname
     :type artname : str
     :param number: number of repeats
     :type number: int
+    :param space: space between characters
+    :type space: int
     :return: None
     """
     try:
         if artname == "UnicodeEncodeError":
             raise UnicodeEncodeError(
                 'test', u"", 42, 43, 'test unicode-encode-error')
         result, artname = art(
-            artname=artname, number=number, __detailed_return=True)
+            artname=artname, number=number, __detailed_return=True,space=space)
```

Comment:
Change it to `artname=artname, number=number, __detailed_return=True, space=space)` and run `autopep8.sh` or `autopep8.bat` for fixing this line.